homeassistant/components/binary_sensor/doorbird.py

```diff
@@ -0,0 +1,83 @@
+"""Support for reading binary states from a DoorBird video doorbell."""
+from datetime import timedelta
+import logging
+import voluptuous as vol
+
+from homeassistant.components.binary_sensor import BinarySensorDevice,\
+    PLATFORM_SCHEMA
+from homeassistant.components.doorbird import DOMAIN
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_MONITORED_CONDITIONS, STATE_UNKNOWN
+from homeassistant.util import Throttle
```

Comment:
Please [order](https://home-assistant.io/developers/development_guidelines/) the import.